tests/tools.vitruv.applications.pcmumlclass.tests/src/tools/vitruv/applications/pcmumlclass/tests/NewPcmUmlClassApplicationTest.xtend

```diff
@@ -0,0 +1,140 @@
+package tools.vitruv.applications.pcmumlclass.tests
+
+import java.nio.file.Path
+import org.eclipse.emf.common.util.URI
+import org.eclipse.emf.ecore.EObject
+import org.eclipse.emf.ecore.resource.ResourceSet
+import org.eclipse.uml2.uml.Model
+import org.eclipse.uml2.uml.UMLFactory
+import org.eclipse.xtend.lib.annotations.Accessors
+import org.junit.jupiter.api.AfterEach
+import org.junit.jupiter.api.BeforeEach
+import org.palladiosimulator.pcm.repository.RepositoryFactory
+import tools.vitruv.applications.pcmumlclass.CombinedPcmToUmlClassReactionsChangePropagationSpecification
+import tools.vitruv.applications.pcmumlclass.CombinedUmlClassToPcmReactionsChangePropagationSpecification
+import tools.vitruv.framework.views.View
+import tools.vitruv.testutils.ViewBasedVitruvApplicationTest
+import tools.vitruv.applications.pcmumlclass.DefaultLiterals
+import static org.hamcrest.MatcherAssert.assertThat
+import static tools.vitruv.testutils.matchers.ModelMatchers.isNoResource
+import static tools.vitruv.testutils.matchers.ModelMatchers.isResource
+
+import static extension tools.vitruv.applications.pcmumlclass.tests.PcmQueryUtil.*
+import static extension tools.vitruv.applications.pcmumlclass.tests.UmlQueryUtil.*
+
+abstract class NewPcmUmlClassApplicationTest extends ViewBasedVitruvApplicationTest {
```

Comment:
I think the name `NewPcmUmlClassApplicationTest` is not ideal, as it does not specify why its new.
Maybe rather something like `ViewBasedPcmUmlClassApplicationTest`?

Reply:
Refactored New to ViewBased and old to Legacy in [cc11817](https://github.com/vitruv-tools/Vitruv-CaseStudies/pull/214/commits/cc11817d10f25564f3e5ab34bff42b514d14bf2d)

Reply:
I would suggest to simply name it `PcmUmlClassApplicationTest`, for the old one the Legacy prefix is good. In the long term, all tests should run view-based, thus we don't need to indicate it here (like it's also not indicated in the already view-based UML <-> Java tests).

Reply:
Renamed in https://github.com/vitruv-tools/Vitruv-CaseStudies/pull/214/commits/fe1fd2240605f1aaca69f8ed3f0468fff8ee740d